### DISCUSSION TITLE
Export link data

### DISCUSSION THREAD

**Discussion by Weiming-Hu** [firefly-iii/firefly-iii#8579] [Firefly III]
Currently, I'm using `php /var/www/firefly/artisan firefly-iii:export-data --export-transactions --token <my_token>` to export transactions and I'm aware of the [numerous options for exporting data](https://docs.firefly-iii.org/tutorials/firefly-iii/exporting-data/#command-line). I'm wondering whether there is a way to export information on linked transactions.

Since linked transactions currently do not do anything, I'm trying to write my own analysis code to handle reimbursement/refund. But I need a way to retain this link information. I don't believe it is present in the result from `--export-transactions`.

Thank you.

**Comment by JC5** [comment]
It is not, no. I'll make of adding *another* export to extract this information from Firefly III.

**Reply by Weiming-Hu** [reply]
Thank you. I currently have a workaround with the v1 API. But I will be happy to test the new function if it is ready.
